griptape/drivers/image_generation/huggingface_diffusion_pipeline_image_generation_driver.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import io
+from abc import ABC
+from typing import Optional
+
+from attrs import define, field
+
+from griptape.artifacts import ImageArtifact
+from griptape.drivers import BaseDiffusionPipelineImageGenerationModelDriver, BaseImageGenerationDriver
+from griptape.utils import import_optional_dependency
+
+
+@define
+class HuggingFaceDiffusionPipelineImageGenerationDriver(BaseImageGenerationDriver, ABC):
```

Comment:
Should this subclass `BaseImageGenerationDriver` instead?

Reply:
I assume you mean `BaseMultiModelImageGenerationDriver`. That was the original intention, but the Pipeline drivers require a substantially different interface than those that inherit from `BaseImageGenerationModelDriver` as required by the `BaseMultiModelImageGenerationDriver`.